galsim/shapelet.py

```diff
@@ -125,31 +128,29 @@ def __init__(self, sigma, order, bvec=None, gsparams=None):
             if len(bvec) != bvec_size:
                 raise ValueError("bvec is the wrong size for the provided order")
             import numpy
-            bvec = LVector(order,numpy.array(bvec))
+            bvec = LVector(order, numpy.array(bvec))
 
         GSObject.__init__(self, _galsim.SBShapelet(sigma, bvec, gsparams))
         self._gsparams = gsparams
 
-    def getSigma(self):
-        return self.SBProfile.getSigma()
+    def getSigma(self): return self.SBProfile.getSigma()
 
-    def getOrder(self):
-        return self.SBProfile.getBVec().order
+    def getOrder(self): return self.SBProfile.getBVec().order
 
-    def getBVec(self):
-        return self.SBProfile.getBVec().array
+    def getBVec(self): return self.SBProfile.getBVec().array
 
     @property
     def sigma(self): return self.getSigma()
+
     @property
     def order(self): return self.getOrder()
+
     @property
     def bvec(self): return self.getBVec()
 
-    def getPQ(self,p,q):
-        return self.SBProfile.getBVec().getPQ(p,q)
-    def getNM(self,N,m):
-        return self.SBProfile.getBVec().getPQ((N+m)/2,(N-m)/2)
+    def getPQ(self, p, q): return self.SBProfile.getBVec().getPQ(p, q)
+
+    def getNM(self, N, m): return self.SBProfile.getBVec().getPQ((N+m)/2, (N-m)/2)
```

Comment:
Again, these don't belong as one-liners.  (unlike the properties)  Leave these the way they were please.